l10n_br_account_product/models/account_invoice.py

```diff
@@ -756,6 +757,21 @@ def _compute_price(self):
     freight_value = fields.Float(
         'Frete', digits=dp.get_precision('Account'), default=0.00)
     fiscal_comment = fields.Text(u'Observação Fiscal')
+    xped = fields.Char(
+        string=u"Código do Pedido (xPed)",
+        size=15,
+    )
+    nitemped = fields.Char(
+        string=u"Item do Pedido (nItemPed)",
+    )
+
+    @api.onchange("nitemped")
```

Comment:
I agree the tag in PySPED might be needed to be changed to `TagInteiro`, in which case we would need to change the assignment code in `document.py` to make sure it's sending an integer if the value is present in the invoice line.

However this is completely unrelated to why I chose to use `field.Char` instead of `field.Integer`: the field is optional, and should be empty when presented to the user, instead of containing a `0`.

If you want, I can add a comment to the code to make this clear.